jax/experimental/host_callback.py

```diff
@@ -1285,12 +1286,12 @@ def _unpack_transform(name, *params):
       # TODO: implement a proper error handling for TPU
       if device.platform != "tpu":
         canonical_flat_results = [xla.canonicalize_dtype(np.arange(12345, dtype=np.int8))]
-        if logging.vlog_is_on(2):
-          logging.vlog(2, f"Outside call consumer {callback} exception {e}. Sending to infeed the error result.")
+        logger.debug("Outside call consumer %s exception %s. Sending to infeed the error result.",
+                     callback, e)
```

Comment:
Shouldn't this be a `logger.error`, since we are logging exceptions here? Same below.

Reply:
I think I'd leave it as is, because this is probably not intended to be seen by the user: note we reraise the exception below.